orte/mca/rtc/hwloc/rtc_hwloc.h

```diff
@@ -45,7 +45,7 @@ typedef struct {
 ORTE_MODULE_DECLSPEC extern orte_rtc_hwloc_component_t mca_rtc_hwloc_component;
 
 extern orte_rtc_base_module_t orte_rtc_hwloc_module;
-
+extern bool orte_report_bindings_use_hwview;
```

Comment:
this externally visible symbol may bother the ABI compatibility crew,  but I'm okay with this.

Reply:
I'm not really wanting it to become an external part of the ABI, it's just used inside the component. But its use spans multiple files within the component.

I don't use mca_base_var_get_value() much, but without putting it in a global I think that would be the only way to access it in the other file